### PR TITLE
Add recipe for censo

### PR DESCRIPTION
**Pull request by awvwgk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by awvwgk** [comment]
@conda-forge/staged-recipes This noarch-Python recipe is ready for review. Please ignore the failure on Windows.

**Comment by BastianZim** [comment]
The windows error is probably from this: https://www.python.org/dev/peps/pep-0597/ same as in https://gitter.im/conda-forge/conda-forge.github.io?at=620111b2708e9c3dd76832f3

Do you want to submit a patch to fix it so that the tests run through? Happy to ignore it as well since it's noarch.

File: recipes/censo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - {{ name }} = censo_qm.censo:main
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
```
Comment (BastianZim) [inline recipes/censo/meta.yaml:24]: There is a list of requirements here: https://github.com/grimme-lab/censo#requirements

Are they not needed?

Comment (awvwgk) [reply recipes/censo/meta.yaml:24]: Added a run requirement on `xtb`.

While Orca is free for academic use, I can't package it here, however the program will search for a user installation and pick it up if available in the `PATH`. I hope that is okay.

The Turbomole, COSMOtherm, cefine dependencies require a commercial license, but is not required for the basic functionality.

**Comment by awvwgk** [comment]
> Do you want to submit a patch to fix it so that the tests run through? Happy to ignore it as well since it's noarch.

I reported it upstream (https://github.com/grimme-lab/CENSO/issues/25). But probably won't install on Windows, since I don't package `xtb` for this platform.

File: recipes/censo/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipes/censo/meta.yaml:24]: Ok that's fine for me. Does it make sense to add them under `run_constrained` or are they not suited for conda (Not necessarily conda-forge) at all?

**Pull request merged**